hera_cal/abscal.py

```diff
@@ -83,34 +81,38 @@ def __init__(self, model, data, refant=None, wgts=None, antpos=None, freqs=None,
 
         Parameters:
         -----------
-        model : visibility data of refence model, type=DataContainer
+        model : Visibility data of refence model, type=dictionary
```

Comment:
Why this change? Are we never using any of DataContainer's special properties?

Reply:
I thought you wanted to deprecate `DataContainer`, so I thought it would just be apt to use a dictionary, which I think is actually the only thing we need, but I can add `or DataContainer` if you planned to keep it around...

Reply:
No, the whole point of `HERAData` is that it spits out DataContainers. We're definitely keeping them.